jib-core/src/test/java/com/google/cloud/tools/jib/registry/RegistryAuthenticatorTest.java

```diff
@@ -50,23 +59,50 @@ public void testFromAuthenticationMethod_bearer()
   }
 
   @Test
-  public void testFromAuthenticationMethod_oauth2()
-      throws RegistryAuthenticationFailedException, IOException {
-    RegistryAuthenticator registryAuthenticator =
-        RegistryAuthenticator.fromAuthenticationMethod(
-            "Bearer realm=\"https://somerealm\",service=\"someservice\",scope=\"somescope\"",
-            registryEndpointRequestProperties);
-    BlobHttpContent blobHttpContent =
-        registryAuthenticator.getOAuth2AuthRequestBody("scope", "sometoken");
-    // the content type should be "application/x-www-form-urlencoded"
-    Assert.assertEquals("application/x-www-form-urlencoded", blobHttpContent.getType());
-    try (ByteArrayOutputStream out = new ByteArrayOutputStream()) {
-      blobHttpContent.writeTo(out);
-      Assert.assertEquals(
-          "grant_type=refresh_token&service=someservice&"
-              + "scope=repository:someimage:scope&refresh_token=sometoken",
-          out.toString());
-    }
+  public void testAuthRequestParameters_basicAuth() {
+    Assert.assertEquals(
+        "service=someservice&scope=repository:someimage:scope",
+        registryAuthenticator.getAuthRequestParameters("scope"));
+  }
+
+  @Test
+  public void testAuthRequestParameters_oauth2() {
+    registryAuthenticator.setCredential(Credential.basic("<token>", "oauth2_access_token"));
```

Comment:
shall we rename the method : Credential.basic ?

Reply:
I just tried this, and it touches too many files. I filed an issue https://github.com/GoogleContainerTools/jib/issues/1526, and I'll create a PR for that soon.